ouroboros-consensus-shelley-test/test/Test/ThreadNet/RealTPraos.hs

```diff
@@ -127,7 +132,7 @@ prop_simple_real_tpraos_convergence TestSetup
           setupK
           setupD
           tpraosSlotLength
-          maxKESEvolution
```

Comment:
Add the following comment to `mkGenesisConfig`: a KES algorithm supports a particular max number of KES evolutions, but we can configure a potentially lower max for the ledger.